src/common/Page.ts

```diff
@@ -1871,7 +1871,9 @@ export class Page extends EventEmitter {
   async reload(options?: WaitForOptions): Promise<HTTPResponse | null> {
     const result = await Promise.all([
       this.waitForNavigation(options),
-      this.#client.send('Page.reload'),
+      this.evaluate(() => {
+        window.location.reload();
```

Comment:
window.location might be overloaded by the page? Why not keep using Page.reload?

Reply:
another thing discovered: `window.location.reload` would not work for data URLs.